axis_AccelX: 1
axis_AccelY: 0
axis_AccelZ: 2
axis_GyroX: 1
axis_GyroY: 0
axis_GyroZ: 2
c0_AccelX: 0.1712259326732673281323827
c1_AccelX: 0.006185200000000000072897244
c0_AccelY: 1.816366653465347091866988
c1_AccelY: -0.006185200000000000072897244
c0_AccelZ: -2.177110661386136847283979
c1_AccelZ: -0.006185200000000000072897244
c0_GyroX: 0.05943721247524752504531875
c1_GyroX: -0.0003196569999999999923609439
c0_GyroY: 0.001126711801980199989525744
c1_GyroY: 0.0003196569999999999923609439
c0_GyroZ: 0.02021751401980198536989519
c1_GyroZ: 0.0003196569999999999923609439
c0_CompassX: 0
c0_CompassY: 0
com_port: /dev/ttyS0
baudrate: 9600
